server/services/mqtt/lib/listenToCustomMqttTopicIfNeeded.js

```diff
@@ -28,7 +28,7 @@ async function listenToCustomMqttTopicIfNeeded(device) {
       object_path: objectPath,
     });
     // Listen to MQTT topic
-    if (this.mqttClient) {
+    if (mqttTopic && mqttTopic.length) {
```

Comment:
Why remove the this.mqttClient ? It's still relevant

Reply:
It's not good to code while doing something else ^^ Sorry